git2consul.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
I opened https://github.com/autopilotpattern/wordpress/issues/4 to attempt a git2consul implementation later.